run_dir/design/flowcell.html

```diff
@@ -152,47 +159,75 @@
                       <th>Mean&nbsp;Quality&nbsp;Score</th>
                     {% end %}
                   </tr>
+                
+                <!--noindex issue-->
                 {% set total_undetermined = 0 %}
-                   <!--noindex issue-->
-                 {% if len(flowcell['lane'][lane_number]) == 1 and flowcell['lane'][lane_number][0]['barcode'] == 'unknown' %}
-                   {% set total_undetermined = flowcell['lane'][lane_number][0]['clustersnb'] %}
-                 {% end %}
+                {% if len(flowcell['lane'][lane_number]) == 1 and flowcell['lane'][lane_number][0]['barcode'] == 'unknown' %}
+                  {% set total_undetermined = flowcell['lane'][lane_number][0]['clustersnb'] %}
+                {% end %}
+                
+                <!--Primary sample table -->
                 {% for sample in flowcell['lane'][lane_number] %}
                   {% set sample_q30 = sample.get('overthirty', 0.0) %}
                   {% try %}{% set sample_q30 = float(sample_q30) %}{% except %}{% end %}
                   <tr>
                     <td>{{ sample.get('Project', '').replace('__', '.') }}</td>
                     <td>{{ sample.get('SampleName') }}</td>
-                    <td class="text-right">{% for thousand in sample.get('yield', '').split(',') %}<span class="thousand_group">{{ thousand }}</span>{% end %}</td>
-                    <td class="text-right 
+                    <td class="text-left">{% for thousand in sample.get('yield', '').split(',') %}<span class="thousand_group">{{ thousand }}</span>{% end %}</td>
```

Comment:
Why did you make this left-aligned? Usually numbers are easier to scan if right-aligned:

---

<p align="right">
283<br>
3,921<br>
1,121<br>
29,383<br>
</p>

---

<p>
283<br>
3,921<br>
1,121<br>
29,383<br>
</p>